docs/proposals/milestones/workflow_project-hierarchy.md

```
@@ -0,0 +1,259 @@
+### New Domain: `src/domains/milestones/`
+
+| Route                                          | Verb       | Purpose                                                                                                                                                                                   |
+| :--------------------------------------------- | :--------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `/projects/:projectId/milestones`              | **GET**    | List all milestones for a project (name, book scope, progress, status badge).                                                                                                             |
+| `/projects/:projectId/milestones`              | **POST**   | Create a new milestone. Receives: name, selected books, type, connectivity profile. Creates `project_units` + `project_unit_bible_books` + `chapter_assignments`. Triggers DBL ingestion. |
+| `/projects/:projectId/milestones/:milestoneId` | **GET**    | Get a single milestone's details (chapters, assignments, progress).                                                                                                                       |
+| `/projects/:projectId/milestones/:milestoneId` | **DELETE** | Delete a milestone (cascades to assignments and translations).                                                                                                                            |
+
+**Domain structure:**
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)